main.go

```diff
@@ -92,14 +89,21 @@ func main() {
 			src, _ := osext.Executable()
 			dest := filepath.Dir(src)
 
-			os.Mkdir(tempToolsPath, 0777)
-			hideFile(tempToolsPath)
+			// Instantiate Tools
+			usr, _ := user.Current()
+			directory := usr.HomeDir + "/.arduino-create"
```

Comment:
Better using https://golang.org/pkg/path/filepath/#Join which uses pathSeparator and not /